ng2-components/ng2-alfresco-documentlist/src/components/document-menu-action.component.ts

```diff
@@ -127,4 +148,25 @@ export class DocumentMenuActionComponent {
     isFolderNameEmpty() {
         return this.folderName === '' ? true : false;
     }
+
+    hasPermission(permission: string): boolean {
+        let hasPermission: boolean = true;
+        if (this.allowableOperations) {
+            let permFound = this.allowableOperations.find(element => element === permission);
+            hasPermission = !permFound && this.disableWithNoPermission ? false : true;
+        } else if (this.disableWithNoPermission) {
```

Comment:
Wrong condition
This method will return true if the user doesn't have any permission (allowableOperation) and the  property disableWithNoPermission is false instead the return should be false
Can you add a unit test?